### PR TITLE
chore(windows): cleanup resources 🎢

### PR DESCRIPTION
**Pull request by mcdurdin**
Moves shared resources to common, Developer resources to developer, and Windows-only resources to windows/src/desktop/branding. Removes old, unused resources, and cleans up references.

This will be most easily reviewed commit-by-commit, because the changes are grouped more neatly this way.

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by ermshiperete** [comment]
`common/resources/icons/filetypes/Thumbs.db` - is this actually used/needed by our code or is that just the Windows Explorer thumbnail database?

**Review by ermshiperete — APPROVED** [review]

File: windows/src/desktop/inst/keymandesktop.wxs [hunk 1]
```diff
@@ -126,11 +126,11 @@
               </Component>
 
               <Component>
-                <File Id="appicon.ico" Name="appicon.ico" Source="..\..\global\res\90\desktop\appicon.ico" KeyPath="yes" />
+                <File Id="appicon.ico" Name="appicon.ico" Source="..\..\desktop\branding\appicon.ico" KeyPath="yes" />
```
[the remark this answers is not in the document]
Comment (mcdurdin) [reply windows/src/desktop/inst/keymandesktop.wxs]: Cleaned up, thanks!

**Comment by mcdurdin** [comment]
> `common/resources/icons/filetypes/Thumbs.db` - is this actually used/needed by our code or is that just the Windows Explorer thumbnail database?

Yeah, it's a Windows Explorer thing... it should be git-ignored globally. Removed and added entry to .gitignore./

**Comment by jahorton** [comment]
New commits look good and clean.

**Pull request merged**